core/src/main/java/hudson/model/Executor.java

```diff
@@ -587,6 +587,17 @@ public boolean isActive() {
     }
 
     /**
+     * Returns {@code true} if the {@code executorCell.jelly} for this {@link Executor} should be visible.
+     *
+     * @return {@code true} if the {@code executorCell.jelly} for this {@link Executor} should be visible.
```

Comment:
I don't like leaving the first sentence blank and I don't like a missing `@return`